x-pack/plugins/apm/server/routes/correlations/utils/get_duration_field.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  SPAN_DURATION,
+  TRANSACTION_DURATION,
+  TRANSACTION_DURATION_HISTOGRAM,
+} from '../../../../common/elasticsearch_fieldnames';
+import { LATENCY_DISTRIBUTION_CHART_TYPE } from '../../../../common/latency_distribution_chart_types';
+
+const {
+  TRANSACTION_DETAILS,
+  LATENCY_CORRELATIONS,
+  FAILED_TRANSACTIONS_CORRELATIONS,
+  DEPENDENCY_LATENCY_DISTRIBUTION,
+} = LATENCY_DISTRIBUTION_CHART_TYPE;
+
```

Comment:
This addresses feedback in the last PR: https://github.com/elastic/kibana/pull/136083#discussion_r917926782. The chart type get passed from the client side when the chart is rendered.